includes/Classifai/Blocks/recommended-content-block/inspector-controls/taxonomy-controls.js

```diff
@@ -112,14 +122,39 @@ const TaxonomyControls = ( { onChange, query } ) => {
 			[]
 		);
 	};
+
+	const UseTermToggle = () => {
+		return (
+			<ToggleControl
+				label={ __(
+					"Use this Post's Categories and Tags",
```

Comment:
Ideally we don't hardcode the word `Post` here, as this could be another type of content (page or CPT)